README.md

```diff
@@ -24,6 +24,34 @@ The purpose of this application is to sync ArchivesSpace records to FOLIO.  It a
 rails folio_sync:run
 ```
 
+## Tasks
+
+### `folio_sync:process_marc_xml`
+This task allows you to test the processing of a MARC XML file for a specific `bib_id`. It reads the MARC XML file from the `tmp/marc_files` directory, processes it, and applies the necessary transformations.
+
+#### Usage:
+```bash
+rake 'folio_sync:process_marc_xml[<bib_id>]'
```

Comment:
Thinking ahead about this app also being the eventual home of Folio-to-Hyacinth sync functionality, would you mind namespacing this and other rake tasks under a `folio_sync:aspace_to_folio:` namespace? (e.g. folio_sync:aspace_to_folio:process_marc_xml)

Reply:
Updated! Also added missing descriptions to new tasks